<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AreaWidget</class>
 <widget class="QWidget" name="AreaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AreaWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <item>
    <widget class="QWidget" name="drawAreaWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>120</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="toolWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="areaNameLabel">
        <property name="text">
         <string>영역 이름</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="areaNameEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="colorButton">
        <property name="text">
         <string>영역 색상 선택</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>삭제</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>저장</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>취소</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>122</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
